fixtures/js-stubs/releaseCounts.js

```diff
@@ -0,0 +1,28 @@
+export function ReleaseCounts() {
```

Comment:
this might need to be TeamReleaseCounts to be more specific. we have a lot of release counts